feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -664,16 +666,14 @@ class BillingWrapperTest {
     }
 
     @Test
-    fun `when querying INAPPs and there is no billing client, don't return anything`() {
+    fun `when querying purchases and there is no billing client, don't return anything`() {
         wrapper = BillingWrapper(mockClientFactory, handler, mockDeviceCache)
-
-        assertThat(wrapper.queryPurchases(BillingClient.SkuType.INAPP)).isNull()
-    }
-
-    @Test
-    fun `when querying SUBs and there is no billing client, don't return anything`() {
-        wrapper = BillingWrapper(mockClientFactory, handler, mockDeviceCache)
-        assertThat(wrapper.queryPurchases(BillingClient.SkuType.SUBS)).isNull()
+        var result: BillingAbstract.QueryPurchasesResult? = null
+        wrapper.queryPurchases("appUserID") {
+            result = it
+        }
+        assertThat(result).isNotNull
```

Comment:
should we check the contents? also, the name of the test method says "don't return anything", which doesn't seem to be accurate anymore